### ISSUE TITLE
Add backlogged messages in the alert stream attachment

### ISSUE THREAD

**Issue by infernix** [Graylog2/graylog-plugin-slack#18]
Since #9 was closed, I'm opening this to keep the backlogged messages on the radar.

Right now there's only two options:
- Send all messages for a given stream to Slack
- Send alerts for new messages to Slack but show no message content

I would like to see an option where some of the backlogged messages in a certain time period are attached to the Slack message. This behaviour would then reproduce what was possible with subscriptions in 0.9.6 - having all backlogged messages for a given stream in a single email.

This is working fine with email alerts, but not with Slack alerts. For example, I'm getting:

```
Alert for Graylog stream MyTestStream​:
Stream had 1 messages in the last 1 minutes with trigger condition more than 0 messages. (Current grace time: 0 minutes)
```

The only details in the Slack message attachments are Stream ID, Stream Title and Stream Description - no message content whatsoever.

**Comment by aberrios85** [comment]
Also seeing this. See example:

```Details:
Stream ID
----------------
56b9f38be4b0acc8fc4f8490

Stream Title
----------------
Asterisk ERROR Messages

Stream Description
----------------
ERROR From an Asterisk Server!```

**Comment by PatrickHuetter** [comment]
+1 for that feature!

**Comment by anc86** [comment]
any chance someone will do this soon please ?

**Comment by robbanp** [comment]
I created a PR for this: 
https://github.com/Graylog2/graylog-plugin-slack/pull/26

**Comment by joschi** [comment]
Fixed by merging #26.
